eclipse-theia/theia
# [plug-in] decorate vscode-ws-jsonrpc to be es5 compatible

#### What it does


fix #6160: decorate vscode-ws-jsonrpc to be es5 compatible, otherwise es5 plugin reader and writer cannot extend them

#### How to test


- remove native node.js debug extension
- install vscode node.js debug extensions
- try to debug mocha tests in Theia repo

You can use any other vscode debug extension, it should not work on master.

#### Review checklist


#### Reminder for reviewers

- as a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)
